packages/react/src/components/ResourceList/ResourceList.jsx

```diff
@@ -158,7 +118,9 @@ class ResourceList extends Component {
             <Fragment>
               <div className={`${prefix}--structured-list-td`}>
                 <strong>{title}</strong>
-                <InlineDiv className={`${prefix}--structured-list-td`}>{description}</InlineDiv>
+                <div className={`${iotPrefix}--inline-div ${prefix}--structured-list-td`}>
```

Comment:
We might wanna name it
`${iotPrefix}--resource-list__inline-div`
where `resource-list` is the iot specific Block and `inline-div` is the element

Reply:
Fixed